pg_search/src/index/directory/mvcc.rs

```diff
@@ -56,7 +54,7 @@ pub enum MvccSatisfies {
 pub struct MVCCDirectory {
     relation_oid: pg_sys::Oid,
     mvcc_style: MvccSatisfies,
-    merge_policy: AllowedMergePolicy,
+    wants_merge: bool,
```

Comment:
See below, but we decided that keeping `AllowedMergePolicy`, rather than using a `bool` will be slightly cleaner.